e2e/features/payment-pending-state-visible-to-tribunal.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Pending Payment state visible to Tribunal
+
+  @payment-pending-state-visibility @RIA-3278
+  Scenario: Pending Payment state visible for Legal Rep
+
+    Given I am signed in as a `Legal Rep`
+    When I go to the `Case List`
+    Then I should see the `State` field
+    And I should see the option `Pending payment` for the `State` field
+
+  @payment-pending-state-visibility @RIA-3278
+  Scenario: Pending payment state visible for Case Officer
+
+    Given I am signed in as a `Case Officer`
+    When I go to the `Case List`
+    Then I should see the `State` field
+    And I should see the option `Pending payment` for the `State` field
+
+  @payment-pending-state-visibility @RIA-3278
+  Scenario: Pending payment state visible for Admin Officer
+
+    Given I am signed in as a `Admin Officer`
+    When I go to the `Case List`
+    Then I should see the `State` field
+    And I should see the option `Pending payment` for the `State` field
+
+  @payment-pending-state-visibility @RIA-3278
+  Scenario: Pending payment state visible for Admin Officer
```

Comment:
I think it should says Judge here, if not it is repetition of line 20

Reply:
Changes name to Judge